newsroom/users/views.py

```diff
@@ -313,11 +313,11 @@ async def edit(args: RouteArguments, params: None, request: Request):
                         updates.pop(field, None)
 
             await UsersService().update(args.user_id, updates)
-            return success_response({"success": True})
+            return Response({"success": True})
 
         return Response(form.errors, 400)
 
-    return success_response(user)
+    return Response(user.to_dict())
```

Comment:
You can now provide a ResourceModel instance directly to Response constructor.
https://github.com/superdesk/superdesk-core/blob/async/superdesk/factory/app.py#L265-L266